app/views/pages/netlify.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application" %>
+    <%= stylesheet_link_tag "application", "data-netlify-preview-style": true  %>
```

Comment:
Yeah, I wanted to have some way to let the Netlify script pick up which styles to use. This is probably overkill since the Netlify route is not using the shared application layout.